frontend/.pa11yci.json

```diff
@@ -0,0 +1,100 @@
+{
+  "defaults": {
+    "timeout": 240000,
+    "runners": ["axe"],
+    "ignore": ["color-contrast"],
+    "concurrency": 1,
+    "chromeLaunchConfig": {
+      "ignoreHTTPSErrors": true,
+      "args": ["--disable-dev-shm-usage", "--no-sandbox"]
+    }
+  },
+  "urls": [
+    {
+      "url": "http://localhost:3000",
+      "viewport": { "width": 320, "height": 480 },
+      "actions": [
+        "wait for element #main-content to be visible",
+        "screen capture screenshots-output/mobile-main-view.png",
+        "click element .usa-navbar button",
+        "screen capture screenshots-output/mobile-expand-menu.png"
+      ]
+    },
+    {
+      "url": "http://localhost:3000",
```

Comment:
Unsure of why we are duplicating each url twice with truncated actions in the bottom one. Is this to take different actions on the second iteration of the URL?